cmd/lotus-shed/verifreg.go

```diff
@@ -268,13 +271,22 @@ var verifRegListClientsCmd = &cli.Command{
 				return err
 			}
 
-			fmt.Printf("%s: %s\n", addr, dcap)
+			fmt.Printf("%s: %s %v\n", addr, dcap, []byte(k))
+
+			keys = append(keys, k)
 
 			return nil
 		}); err != nil {
 			return err
 		}
 
+		for _, k := range keys {
```

Comment:
Did you mean to add this (and if so, do you wanna add it to list-verifiers too)?